### PR TITLE
Fix issue 1

### PR DESCRIPTION
**Pull request by breaman**
I think I got the first part of the assignment done.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -354,3 +354,8 @@ healthchecksdb
 MigrationBackup/
 
 # End of https://www.gitignore.io/api/csharp
+
+# Mac specific files
+.DS_Store
+
+*.sqlite
```
Comment (Keboo) [inline .gitignore]: Shouldn't these changes be on master?

**Comment by Keboo** [comment]
This PR is going against master. Shouldn't it be targeting the Assignment 1 branch?

File: .gitignore (hunk 1 above)
Comment (breaman) [reply .gitignore]: Yes you are correct, thank you for pointing out my idiocracy in front of the whole class. :)

Comment (Keboo) [reply .gitignore]: Sorry, PRs are like mtn dew; when I see one, it is hard to resist.

**Pull request closed without merging**